packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -70,22 +66,22 @@ class ProjectGet(OutputSchema):
     uuid: ProjectID
     name: str
     description: str
-    thumbnail: HttpUrlWithCustomMinLength | Literal[""]
+    thumbnail: HttpUrl | Literal[""]
     creation_date: DateTimeStr
     last_change_date: DateTimeStr
     workbench: NodesDict
     prj_owner: LowerCaseEmailStr
     access_rights: dict[GroupIDStr, AccessRights]
     tags: list[int]
     classifiers: list[ClassifierID] = []
-    state: ProjectState | None
-    ui: EmptyModel | StudyUI | None
+    state: ProjectState | None = None
+    ui: EmptyModel | StudyUI | None = None
     quality: dict[str, Any] = {}
```

Comment:
@pcrespov If I revert the `= None` for the fields in this model, the [create-project-schema](https://github.com/giancarloromeo/osparc-simcore/blob/2c18b4f974852362c33b26636d9b5a8ac8f8acd5/packages/models-library/tests/test_api_schemas_webserver_projects.py#L36) test fails because the mocked response payload doesn't contain them. Are these fields optional?

Reply:
IMO this case a default `=None` makes sense

These mocks were generated using **real** payload from the front-end so it is important that they pass since there is no guarantee that the front-end always follows the openapi specs in the web-api.
Also, as you can see in `services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml` that these fields are not required:
![image](https://github.com/user-attachments/assets/e2c4a725-20b6-4a29-acbf-726ddc57a257)


BTW, once all the tests in the `models_library` pass, you might want to try to re-generate  `services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml` and see if the changes make sense. In principle only the new nullable fields should be update and the rest should be the same. Pay special attention to `required` fields for instance ... 
For that you can  use 
```
cd web/server
make openapi-specs
```